### ISSUE TITLE
2025-02-18, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#87]
# Today's Arxiv Papers

(1) [Idiosyncrasies in Large Language Models](http://arxiv.org/abs/2502.12150v1), 2025-02-17

(2) [HARBOR: Exploring Persona Dynamics in Multi-Agent Competition](http://arxiv.org/abs/2502.12149v1), 2025-02-17

(3) [HermesFlow: Seamlessly Closing the Gap in Multimodal Understanding and Generation](http://arxiv.org/abs/2502.12148v1), 2025-02-17

(4) [Fast or Better? Balancing Accuracy and Cost in Retrieval-Augmented Generation with Flexible User Control](http://arxiv.org/abs/2502.12145v1), 2025-02-17

(5) [Small Models Struggle to Learn from Strong Reasoners](http://arxiv.org/abs/2502.12143v1), 2025-02-17

(6) [SoftCoT: Soft Chain-of-Thought for Efficient Reasoning with LLMs](http://arxiv.org/abs/2502.12134v1), 2025-02-17

(7) [Transformer Dynamics: A neuroscientific approach to interpretability of large language models](http://arxiv.org/abs/2502.12131v1), 2025-02-17

(8) [Scaling Autonomous Agents via Automatic Reward Modeling And Planning](http://arxiv.org/abs/2502.12130v1), 2025-02-17

(9) [Minimal Ranks, Maximum Confidence: Parameter-efficient Uncertainty Quantification for LoRA](http://arxiv.org/abs/2502.12122v1), 2025-02-17

(10) [LLMs on the Line: Data Determines Loss-to-Loss Scaling Laws](http://arxiv.org/abs/2502.12120v1), 2025-02-17

(11) [PRISM: Self-Pruning Intrinsic Selection Method for Training-Free Multimodal Data Selection](http://arxiv.org/abs/2502.12119v1), 2025-02-17

(12) [Scaling Test-Time Compute Without Verification or RL is Suboptimal](http://arxiv.org/abs/2502.12118v1), 2025-02-17

(13) [SWE-Lancer: Can Frontier LLMs Earn $1 Million from Real-World Freelance Software Engineering?](http://arxiv.org/abs/2502.12115v1), 2025-02-17

(14) [A-MEM: Agentic Memory for LLM Agents](http://arxiv.org/abs/2502.12110v1), 2025-02-17

(15) [Personality Structured Interview for Large Language Model Simulation in Personality Research](http://arxiv.org/abs/2502.12109v1), 2025-02-17

(16) [Relational Norms for Human-AI Cooperation](http://arxiv.org/abs/2502.12102v1), 2025-02-17

(17) [Token Communications: A Unified Framework for Cross-modal Context-aware Semantic Communications](http://arxiv.org/abs/2502.12096v1), 2025-02-17

(18) [Meta-Statistical Learning: Supervised Learning of Statistical Inference](http://arxiv.org/abs/2502.12088v1), 2025-02-17

(19) [APB: Accelerating Distributed Long-Context Inference by Passing Compressed Context Blocks across GPUs](http://arxiv.org/abs/2502.12085v1), 2025-02-17

(20) [Can LLMs Simulate Social Media Engagement? A Study on Action-Guided Response Generation](http://arxiv.org/abs/2502.12073v1), 2025-02-17

(21) [TokenSkip: Controllable Chain-of-Thought Compression in LLMs](http://arxiv.org/abs/2502.12067v1), 2025-02-17

(22) [CONSTRUCTA: Automating Commercial Construction Schedules in Fabrication Facilities with Large Language Models](http://arxiv.org/abs/2502.12066v1), 2025-02-17

(23) [Formalizing Complex Mathematical Statements with LLMs: A Study on Mathematical Definitions](http://arxiv.org/abs/2502.12065v1), 2025-02-17

(24) [AI-generated Text Detection with a GLTR-based Approach](http://arxiv.org/abs/2502.12064v1), 2025-02-17

(25) [Designing Role Vectors to Improve LLM Inference Behaviour](http://arxiv.org/abs/2502.12055v1), 2025-02-17

(26) [PhysReason: A Comprehensive Benchmark towards Physics-Based Reasoning](http://arxiv.org/abs/2502.12054v1), 2025-02-17

(27) [A Dual-Perspective NLG Meta-Evaluation Framework with Automatic Benchmark and Better Interpretability](http://arxiv.org/abs/2502.12052v1), 2025-02-17

(28) [A Survey on Bridging EEG Signals and Generative AI: From Image and Text to Beyond](http://arxiv.org/abs/2502.12048v1), 2025-02-17

(29) [KnowPath: Knowledge-enhanced Reasoning via LLM-generated Inference Paths over Knowledge Graphs](http://arxiv.org/abs/2502.12029v1), 2025-02-17

(30) [SafeChain: Safety of Language Models with Long Chain-of-Thought Reasoning Capabilities](http://arxiv.org/abs/2502.12025v1), 2025-02-17

(31) [Teaching LLMs According to Their Aptitude: Adaptive Reasoning for Mathematical Problem Solving](http://arxiv.org/abs/2502.12022v1), 2025-02-17

(32) [Atom of Thoughts for Markov LLM Test-Time Scaling](http://arxiv.org/abs/2502.12018v1), 2025-02-17

(33) [Presumed Cultural Identity: How Names Shape LLM Responses](http://arxiv.org/abs/2502.11995v1), 2025-02-17

(34) [Generating Text from Uniform Meaning Representation](http://arxiv.org/abs/2502.11973v1), 2025-02-17

(35) [Navigating the Helpfulness-Truthfulness Trade-Off with Uncertainty-Aware Instruction Fine-Tuning](http://arxiv.org/abs/2502.11962v1), 2025-02-17

(36) [Can Your Uncertainty Scores Detect Hallucinated Entity?](http://arxiv.org/abs/2502.11948v1), 2025-02-17

(37) [On Representational Dissociation of Language and Arithmetic in Large Language Models](http://arxiv.org/abs/2502.11932v1), 2025-02-17

(38) [BRIGHTER: BRIdging the Gap in Human-Annotated Textual Emotion Recognition Datasets for 28 Languages](http://arxiv.org/abs/2502.11926v1), 2025-02-17

(39) [GRAPHGPT-O: Synergistic Multimodal Comprehension and Generation on Graphs](http://arxiv.org/abs/2502.11925v1), 2025-02-17

(40) [From Text to Trust: Empowering AI-assisted Decision Making with Adaptive LLM-powered Analysis](http://arxiv.org/abs/2502.11919v1), 2025-02-17

(41) [EssayJudge: A Multi-Granular Benchmark for Assessing Automated Essay Scoring Capabilities of Multimodal Large Language Models](http://arxiv.org/abs/2502.11916v1), 2025-02-17

(42) [Adversarial Alignment for LLMs Requires Simpler, Reproducible, and More Measurable Objectives](http://arxiv.org/abs/2502.11910v1), 2025-02-17

(43) [MMRC: A Large-Scale Benchmark for Understanding Multimodal Large Language Model in Real-World Conversation](http://arxiv.org/abs/2502.11903v1), 2025-02-17

(44) [CAMEL: Continuous Action Masking Enabled by Large Language Models for Reinforcement Learning](http://arxiv.org/abs/2502.11896v1), 2025-02-17

(45) [Continual Quantization-Aware Pre-Training: When to transition from 16-bit to 1.58-bit pre-training for BitNet language models?](http://arxiv.org/abs/2502.11895v1), 2025-02-17

(46) [LIMR: Less is More for RL Scaling](http://arxiv.org/abs/2502.11886v1), 2025-02-17

(47) [Leveraging Dual Process Theory in Language Agent Framework for Real-time Simultaneous Human-AI Collaboration](http://arxiv.org/abs/2502.11882v1), 2025-02-17

(48) [Hypothesis-Driven Theory-of-Mind Reasoning for Large Language Models](http://arxiv.org/abs/2502.11881v1), 2025-02-17

(49) [Bitnet.cpp: Efficient Edge Inference for Ternary LLMs](http://arxiv.org/abs/2502.11880v1), 2025-02-17

(50) [JoLT: Joint Probabilistic Predictions on Tabular Data Using LLMs](http://arxiv.org/abs/2502.11877v1), 2025-02-17

(51) [Southern Newswire Corpus: A Large-Scale Dataset of Mid-Century Wire Articles Beyond the Front Page](http://arxiv.org/abs/2502.11866v1), 2025-02-17

(52) [FedEAT: A Robustness Optimization Framework for Federated LLMs](http://arxiv.org/abs/2502.11863v1), 2025-02-17

(53) [Understanding In-Context Machine Translation for Low-Resource Languages: A Case Study on Manchu](http://arxiv.org/abs/2502.11862v1), 2025-02-17

(54) [Exploring Large Language Models in Healthcare: Insights into Corpora Sources, Customization Strategies, and Evaluation Metrics](http://arxiv.org/abs/2502.11861v1), 2025-02-17

(55) [LLMs as a synthesis between symbolic and continuous approaches to language](http://arxiv.org/abs/2502.11856v1), 2025-02-17

(56) [StructTransform: A Scalable Attack Surface for Safety-Aligned Large Language Models](http://arxiv.org/abs/2502.11853v1), 2025-02-17

(57) [BaxBench: Can LLMs Generate Correct and Secure Backends?](http://arxiv.org/abs/2502.11844v1), 2025-02-17

(58) [Can LLM Agents Maintain a Persona in Discourse?](http://arxiv.org/abs/2502.11843v1), 2025-02-17

(59) [Model Generalization on Text Attribute Graphs: Principles with Large Language Models](http://arxiv.org/abs/2502.11836v1), 2025-02-17

(60) [HAAN: A Holistic Approach for Accelerating Normalization Operations in Large Language Models](http://arxiv.org/abs/2502.11832v1), 2025-02-17

(61) [Intuitive physics understanding emerges from self-supervised pretraining on natural videos](http://arxiv.org/abs/2502.11831v1), 2025-02-17

(62) [Text Classification in the LLM Era - Where do we stand?](http://arxiv.org/abs/2502.11830v1), 2025-02-17

(63) [Code-Vision: Evaluating Multimodal LLMs Logic Understanding and Code Generation Capabilities](http://arxiv.org/abs/2502.11829v1), 2025-02-17

(64) [M-ABSA: A Multilingual Dataset for Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2502.11824v1), 2025-02-17

(65) [Towards Understanding Fine-Tuning Mechanisms of LLMs via Circuit Analysis](http://arxiv.org/abs/2502.11812v1), 2025-02-17

(66) [FineFilter: A Fine-grained Noise Filtering Mechanism for Retrieval-Augmented Large Language Models](http://arxiv.org/abs/2502.11811v1), 2025-02-17

(67) [Exploring Translation Mechanism of Large Language Models](http://arxiv.org/abs/2502.11806v1), 2025-02-17

(68) [Table-Critic: A Multi-Agent Framework for Collaborative Criticism and Refinement in Table Reasoning](http://arxiv.org/abs/2502.11799v1), 2025-02-17

(69) [Personality Editing for Language Models through Relevant Knowledge Editing](http://arxiv.org/abs/2502.11789v1), 2025-02-17

(70) [Efficient Response Generation Method Selection for Fine-Tuning Large Language Models](http://arxiv.org/abs/2502.11779v1), 2025-02-17

(71) [video-SALMONN-o1: Reasoning-enhanced Audio-visual Large Language Model](http://arxiv.org/abs/2502.11775v1), 2025-02-17

(72) [The Validation Gap: A Mechanistic Analysis of How Language Models Compute Arithmetic but Fail to Validate It](http://arxiv.org/abs/2502.11771v1), 2025-02-17

(73) [Cognitive-Aligned Document Selection for Retrieval-augmented Generation](http://arxiv.org/abs/2502.11770v1), 2025-02-17

(74) [From Selection to Generation: A Survey of LLM-based Active Learning](http://arxiv.org/abs/2502.11767v1), 2025-02-17

(75) [Warmup-Distill: Bridge the Distribution Mismatch between Teacher and Student before Knowledge Distillation](http://arxiv.org/abs/2502.11766v1), 2025-02-17

(76) [HintsOfTruth: A Multimodal Checkworthiness Detection Dataset with Real and Synthetic Claims](http://arxiv.org/abs/2502.11753v1), 2025-02-17

(77) [Language Models Can See Better: Visual Contrastive Decoding For LLM Multimodal Reasoning](http://arxiv.org/abs/2502.11751v1), 2025-02-17

(78) [SQL-o1: A Self-Reward Heuristic Dynamic Search Method for Text-to-SQL](http://arxiv.org/abs/2502.11741v1), 2025-02-17

(79) [ReviewEval: An Evaluation Framework for AI-Generated Reviews](http://arxiv.org/abs/2502.11736v1), 2025-02-17

(80) [MT-RAIG: Novel Benchmark and Evaluation Framework for Retrieval-Augmented Insight Generation over Multiple Tables](http://arxiv.org/abs/2502.11735v1), 2025-02-17

(81) [Plant in Cupboard, Orange on Table, Book on Shelf. Benchmarking Practical Reasoning and Situation Modelling in a Text-Simulated Situated Environment](http://arxiv.org/abs/2502.11733v1), 2025-02-17

(82) [Energy-Conscious LLM Decoding: Impact of Text Generation Strategies on GPU Energy Consumption](http://arxiv.org/abs/2502.11723v1), 2025-02-17

(83) [Enhancing Recommendation Explanations through User-Centric Refinement](http://arxiv.org/abs/2502.11721v1), 2025-02-17

(84) [Can you pass that tool?: Implications of Indirect Speech in Physical Human-Robot Collaboration](http://dx.doi.org/10.1145/3706598.3713780), 2025-02-17

(85) [Ad-hoc Concept Forming in the Game Codenames as a Means for Evaluating Large Language Models](http://arxiv.org/abs/2502.11707v1), 2025-02-17

(86) [LLM Agents Making Agent Tools](http://arxiv.org/abs/2502.11705v1), 2025-02-17

(87) [CMQCIC-Bench: A Chinese Benchmark for Evaluating Large Language Models in Medical Quality Control Indicator Calculation](http://arxiv.org/abs/2502.11703v1), 2025-02-17

(88) [Improve LLM-as-a-Judge Ability as a General Ability](http://arxiv.org/abs/2502.11689v1), 2025-02-17

(89) [MathFimer: Enhancing Mathematical Reasoning by Expanding Reasoning Steps through Fill-in-the-Middle Task](http://arxiv.org/abs/2502.11684v1), 2025-02-17

(90) [RIDE: Enhancing Large Language Model Alignment through Restyled In-Context Learning Demonstration Exemplars](http://arxiv.org/abs/2502.11681v1), 2025-02-17

(91) [Exploring LLM-based Student Simulation for Metacognitive Cultivation](http://arxiv.org/abs/2502.11678v1), 2025-02-17

(92) [Towards Fully Exploiting LLM Internal States to Enhance Knowledge Boundary Perception](http://arxiv.org/abs/2502.11677v1), 2025-02-17

(93) [Diversity-Oriented Data Augmentation with Large Language Models](http://arxiv.org/abs/2502.11671v1), 2025-02-17

(94) [VRoPE: Rotary Position Embedding for Video Large Language Models](http://arxiv.org/abs/2502.11664v1), 2025-02-17

(95) [An Innovative Brain-Computer Interface Interaction System Based on the Large Language Model](http://arxiv.org/abs/2502.11659v1), 2025-02-17

(96) [Competing LLM Agents in a Non-Cooperative Game of Opinion Polarisation](http://arxiv.org/abs/2502.11649v1), 2025-02-17

(97) [DELMAN: Dynamic Defense Against Large Language Model Jailbreaking with Model Editing](http://arxiv.org/abs/2502.11647v1), 2025-02-17

(98) [Deviation Ratings: A General, Clone-Invariant Rating Method](http://arxiv.org/abs/2502.11645v1), 2025-02-17

(99) [Assessing Correctness in LLM-Based Code Generation via Uncertainty Estimation](http://arxiv.org/abs/2502.11620v1), 2025-02-17

(100) [Is Human-Like Text Liked by Humans? Multilingual Human Detection and Preference Against AI](http://arxiv.org/abs/2502.11614v1), 2025-02-17
